packages.json

```diff
@@ -12847,17 +12847,21 @@
   },
   {
     "name": "result",
-    "url": "https://github.com/arnetheduck/nim-result",
+    "alias": "results"
+  },
+  {
+    "name": "results",
+    "url": "https://github.com/arnetheduck/nim-results",
     "method": "git",
     "tags": [
       "library",
       "result",
       "errors",
       "functional"
```

Comment:
Just a thought - maybe also add the tag `option` so there's a higher possibility of people finding the package?

Reply:
6e9e61c8fe8906c350ace4038836be22ef05ca39